testing/trino-server-dev/etc/catalog/delta.properties

```diff
@@ -0,0 +1,22 @@
+#
+# WARNING
+# ^^^^^^^
+# This configuration file is for development only and should NOT be used
+# in production. For example configuration, see the Trino documentation.
+#
+
```

Comment:
useless, remove (i know, we have this in some other fails, but that's useless anyway)

Reply:
This is not useless. We added it so that people wouldn't copy it as a production config. If you feel strongly, we can discuss removing it from all of them.